### ISSUE TITLE
Response box resizable

### ISSUE THREAD

**Issue by albertotn** [mirkoperillo/resting#129]
## Expected Behavior

On a screen with high resolution, there is a lot of space. Resting should display all response details or let user to resize response box at will.

## Actual Behavior

There is no way to resize response

## Steps to Reproduce the Problem

  1. make a call to a REST service with "big" response
  1. watch response box, is not resizable and does not expand on content, see example

![immagine](https://user-images.githubusercontent.com/12526457/97844464-5b532500-1ceb-11eb-853e-5e2cf5a5f680.png)

## Specifications

  - Addon Version: 1.0.3
  - Firefox Version: 82.0.1
  - OS: Windows 10

**Comment by mirkoperillo** [comment]
@albertotn your are right.
Currently Resting shows a vertical scrollbar when the response is big.

Add the possibility to resize the response box can be a good enhancement
Thank you for your suggestion

**Comment by mirkoperillo** [comment]
@albertotn can this PR #131 resolve the issue ?

**Comment by albertotn** [comment]
Not quite, what I have in mind is a resizable space by the user. If you try to write a new comment here on github you can notice it on bottom right of the edit box

**Comment by mirkoperillo** [comment]
Ok, I understand
